components/playlist/browser/playlist_service.cc

```diff
@@ -720,10 +765,7 @@ bool PlaylistService::ShouldGetMediaFromBackgroundWebContents(
 
   CHECK(contents);
   const auto& url = contents->GetVisibleURL();
-
-  return ShouldUseFakeUA(url) ||
-         download_request_manager_->media_detector_component_manager()
-             ->ShouldHideMediaSrcAPI(url);
+  return ShouldGetMediaFromBackgroundWebContents(url);
```

Comment:
I'm not seeing these functions (either of `ShouldGetMediaFromBackgroundWebContents()`) being used anywhere.

Reply:
We have call sites in playlist_render_frame_observer_browsertest.cc